### ISSUE TITLE
Generalization of long-range HFX fraction

### ISSUE THREAD

**Issue by vanderhe** [dftbplus/dftbplus#1490]
For purely long-range corrected hybrids (LC), the CAM parameter $\beta=1$ is hardcoded. Before releasing 24.2 we have to generalize the SK-file extra-tag to allow for arbitrary $0 < \beta \leq 1$.

**Comment by bhourahine** [comment]
@vanderhe is this still a live issue?

**Comment by vanderhe** [comment]
> @vanderhe is this still a live issue?

@bhourahine Yes, it is. We have to decide, if we want to generalize the SK extra tag of LC-type functionals.

**Comment by vanderhe** [comment]
As we are deprecating the `LC` tag in favor of the more general `CAM` tag, this issue is effectively resolved by #1607.
